src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1664,7 +1664,10 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     void TermControl::_CursorPositionChanged(const IInspectable& /*sender*/,
                                              const IInspectable& /*args*/)
     {
-        _tsfTryRedrawCanvas->Run();
+        if (_tsfTryRedrawCanvas)

```

Comment:
this seems sus - why is this in this PR? I don't doubt that this makes sense, but I'm just wondering. I suppose this is possible to hit if we've started closing, and we've `.reset()` `_tsfTryRedrawCanvas` in `Close`, but we're not fully dtor'd yet.

for the record - leave this in. Just curious to see it here.

Reply:
It was crashing on me while I was doing things so I guarded it.